All test fails going native

Test works perfect in jvm mode - fails in native mode

````
org.example.MainTests > printsVersion() FAILED


Failures (1):
  JUnit Jupiter:MainTests:printsVersion()
    MethodSource [className = 'org.example.MainTests', methodName = 'printsVersion', methodParameterTypes = '']
    => picocli.CommandLine$InitializationException: Cannot instantiate org.example.picocli.PropertiesFileVersionProvider: the class has no constructor
       picocli.CommandLine$DefaultFactory.create(CommandLine.java:5685)
       picocli.CommandLine$DefaultFactory.createVersionProvider(CommandLine.java:5671)
       picocli.CommandLine$Model$CommandSpec.updateVersionProvider(CommandLine.java:7487)
       picocli.CommandLine$Model$CommandSpec.updateCommandAttributes(CommandLine.java:7452)
       picocli.CommandLine$Model$CommandReflection.extractCommandSpec(CommandLine.java:11801)
       [...]
     Caused by: java.lang.NoSuchMethodException: org.example.picocli.PropertiesFileVersionProvider.<init>()
       java.lang.Class.getConstructor0(DynamicHub.java:3585)
       java.lang.Class.getDeclaredConstructor(DynamicHub.java:2754)
       picocli.CommandLine$DefaultFactory.create(CommandLine.java:5656)
       picocli.CommandLine$DefaultFactory.create(CommandLine.java:5683)
       [...]
````

My setup
````
java -version
openjdk version "17.0.4" 2022-07-19
OpenJDK Runtime Environment GraalVM CE 22.2.0 (build 17.0.4+8-jvmci-22.2-b06)
OpenJDK 64-Bit Server VM GraalVM CE 22.2.0 (build 17.0.4+8-jvmci-22.2-b06, mixed mode, sharing)
````

````
$ uname --all
Linux fedora 6.1.6-200.fc37.x86_64  Sat Jan 14 16:55:06 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
````

--- a/pom.xml
+++ b/pom.xml
@@ -75,6 +75,24 @@
 	<build>
 		<defaultGoal>exec:java@run</defaultGoal>
 		<plugins>
+			<!-- add picocli annotation processing https://picocli.info/#_annotation_processor -->
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>${maven-compiler-plugin.version}</version>
+				<configuration>
+					<annotationProcessorPaths>
+						<path>
+							<groupId>info.picocli</groupId>
+							<artifactId>picocli-codegen</artifactId>
+							<version>${picocli.version}</version>
+						</path>
+					</annotationProcessorPaths>
+					<compilerArgs>
+						<arg>-Aproject=${project.groupId}/${project.artifactId}</arg>
+					</compilerArgs>
+				</configuration>
+			</plugin>
 			<plugin>
 				<groupId>org.codehaus.mojo</groupId>
 				<artifactId>exec-maven-plugin</artifactId>
@@ -185,24 +203,6 @@
 			<id>native</id>
 			<build>
 				<plugins>
-					<!-- add picocli annotation processing https://picocli.info/#_annotation_processor -->
-					<plugin>
-						<groupId>org.apache.maven.plugins</groupId>
-						<artifactId>maven-compiler-plugin</artifactId>
-						<version>${maven-compiler-plugin.version}</version>
-						<configuration>
-							<annotationProcessorPaths>
-								<path>
-									<groupId>info.picocli</groupId>
-									<artifactId>picocli-codegen</artifactId>
-									<version>${picocli.version}</version>
-								</path>
-							</annotationProcessorPaths>
-							<compilerArgs>
-								<arg>-Aproject=${project.groupId}/${project.artifactId}</arg>
-							</compilerArgs>
-						</configuration>
-					</plugin>
 					<plugin>
 						<groupId>org.graalvm.buildtools</groupId>
 						<artifactId>native-maven-plugin</artifactId>

